packages/jaeger-ui/src/types/config.tsx

```diff
@@ -46,6 +46,10 @@ export type Config = {
   scripts?: TScript[];
   topTagPrefixes?: string[];
   tracking?: {
+    cookieToDimension?: {
```

Comment:
perhaps this should be an array of mappings, so that it can be easily extended to other cookies (e.g. user's org, if we were able to get it).

Reply:
I was on the fence about it but just updated it.